android/app/src/main/java/com/reactnativenavigation/activities/BottomTabActivity.java

```diff
@@ -65,6 +72,20 @@ public void run() {
         });
     }
 
+    protected void setupDrawer(Drawer drawer, Screen screen) {
```

Comment:
No problem at all, it was a blast to work with!  I'll get on that update later today.